neres/session.py

```diff
@@ -0,0 +1,38 @@
+import requests
+
+
+class Session(requests.Session):
+
+    def _store_xsrf_token(self, response):
+        xsrf_token = response.cookies.get('XSRF-TOKEN')
+        if xsrf_token:
+            self.xsrf_token = xsrf_token
+
+    def _set_xsrf_headers(self, kwargs):
+        xsrf_header = {
+            'X-XSRF-TOKEN': self.xsrf_token,
+        }
+        if 'headers' in kwargs:
+            kwargs['headers'].update(xsrf_header)
+        else:
+            kwargs['headers'] = xsrf_header
```

Comment:
If headers are already present I want to amend the xsrf token and not replace everything